ui/components/app/confirm-subtitle/confirm-subtitle.js

```diff
@@ -5,7 +5,7 @@ import { useSelector } from 'react-redux';
 import { SECONDARY } from '../../../helpers/constants/common';
 import {
   Color,
-  FONT_WEIGHT,
+  FontWeight,
```

Comment:
We could actually remove this because the font-weight for bodyMd is already normal